bench/wtperf/wtperf_run_py/perf_stat_collection.py

```diff
@@ -38,23 +38,85 @@ def create_test_stat_path(test_home_path: str, test_stats_file: str):
 
 
 class PerfStatCollection:
-    def __init__(self):
-        self.perf_stats = {}
-
-    def add_stat(self, perf_stat: PerfStat):
-        self.perf_stats[perf_stat.short_label] = perf_stat
-
-    def find_stats(self, test_home: str, operations: List[str]):
-        for stat in self.perf_stats.values():
-            if not operations or stat.short_label in operations:
-                test_stat_path = create_test_stat_path(test_home, stat.stat_file)
-                values = stat.find_stat(test_stat_path=test_stat_path)
-                stat.add_values(values=values)
-
-    def to_value_list(self, brief: bool):
-        stats_list = []
-        for stat in self.perf_stats.values():
-            if not stat.are_values_all_zero():
-                stat_list = stat.get_value_list(brief = brief)
-                stats_list.extend(stat_list)
-        return stats_list
+    def __init__(self, operations: List[str]):
+        self.to_report: List[PerfStat] = []
+        if operations:
+            self.to_report = [stat for stat in self.all_stats() if stat.short_label in operations]
```

Comment:
What if someone running the script passes an invalid stat/operation?

Reply:
Good point! An invalid operation will be silently ignored here.
I've added a validation check in 5393090, and moved the perf_stats constructors in 130b4ae

Reply:
I suggested moving `perf_stats = PerfStatCollection(content["operations"])` before running the test, as I thought we would validate from within the constructor itself. However,  [5393090](https://github.com/wiredtiger/wiredtiger/commit/53930909bd3bebf75f880c74a74ae8e17a931c35) change looks like a better approach.